openlibrary/plugins/openlibrary/js/validate.js

```diff
@@ -29,6 +29,16 @@ export default function initValidate() {
     'Are you sure that\'s the published date?'
     );
 
+    // validate title to make sure it contains at least one non-whitespace
+    // character (otherwise it will appear blank)
+    // used in templates/books/add.html
+    jQuery.validator.addMethod('title', function(value) {
```

Comment:
Oh I was thinking you might be able to put this directly in the html where the title input element appears. Adding a `pattern=".*\S+.*"` to the element should work and won't require any JS

Reply:
Oh sorry! Didn't know you could put regex directly in the input field like that, will change.

Reply:
@cdrini Hello, sorry to take up more of your time. I noticed that jquery validate automatically adds the `novalidate` attribute to the form, which disables the use of `pattern`. Wondering if/how I should be bypassing that? Otherwise I think that route works well.